addons/mrp/models/stock_move.py

```diff
@@ -282,7 +282,7 @@ def split_move_lot(self):
         self.ensure_one()
         view = self.env.ref('mrp.view_stock_move_lots')
         serial = (self.has_tracking == 'serial')
-        only_create = False  # Check picking type in theory
+        only_create = False  # Check operation type in theory
```

Comment:
in comments of code, developers and real functionals should keep talking about picking types.